assemblies/common-resources/src/main/resources-filtered/etc/org.apache.karaf.features.xml

```diff
@@ -60,11 +60,11 @@
 
         <!-- [PPP-4108] Use of vulnerable component jackson-mapper-asl-1.9.2.jar CVE-2017-7525, CVE-2017-15095, CVE-2017-15095 -->
         <bundle
-            originalUri="mvn:org.codehaus.jackson/jackson-core-asl/[1.5.0,${codehaus-jackson.version})"
-            replacement="mvn:org.codehaus.jackson/jackson-core-asl/${codehaus-jackson.version}" mode="maven" />
+            originalUri="mvn:org.codehaus.jackson/jackson-core-asl/[1.5.0,1.9.14)"
+            replacement="mvn:com.fasterxml.jackson.core/jackson-core/${fasterxml-jackson}" mode="maven" />
         <bundle
-            originalUri="mvn:org.codehaus.jackson/jackson-mapper-asl/[1.5.0,${codehaus-jackson.version})"
-            replacement="mvn:org.codehaus.jackson/jackson-mapper-asl/${codehaus-jackson.version}" mode="maven" />
+            originalUri="mvn:org.codehaus.jackson/jackson-mapper-asl/[1.5.0,1.9.14)"
```

Comment:
Similar case to the previous comment.

Reply:
https://mvnrepository.com/artifact/org.codehaus.jackson/jackson-mapper-asl - org.codehaus.jackson/jackson-mapper-asl was moved to com.fasterxml.jackson.core/jackson-databind